cartography/models/ontology/mapping/data/users.py

```diff
@@ -43,6 +43,17 @@
             query="MATCH (u:User)-[:HAS_ACCOUNT]->(:UserAccount)-[:OWNS|HAS]->(k:APIKey) MERGE (u)-[r:OWNS]->(k) ON CREATE SET r.firstseen = timestamp() SET r.lastupdated = $UPDATE_TAG",
             iterative=False,
         ),
+        OntologyRelMapping(
+            __comment__="Link User to ThirdPartyApp",
+            query="MATCH (u:User)-[:HAS_ACCOUNT]->(:UserAccount)-[or:AUTHORIZED|APPLICATION]->(a:ThirdPartyApp) MERGE (u)-[r:AUTHORIZED]->(a) ON CREATE SET r.firstseen = timestamp() SET r.lastupdated = $UPDATE_TAG, r.scopes = coalesce(or.scopes, [])",
```

Comment:
`[or:AUTHORIZED|APPLICATION]`
I'm scared to name this rel "or" given that OR is a reserved word in cypher, maybe we can use something else just to be super super safe?